Looped/Interface/GifCell.swift

```diff
@@ -1,11 +1,27 @@
 import UIKit
+import SwiftyGif
+
+protocol GifPlayable {
+    func openGifFrom(url: URL)
+}
 
 final class GifCell: TableViewCell {
     
+    var gifUrlString: String? {
+        didSet {
+            if let gifUrl = URL(string: gifUrlString ?? "") {
```

Comment:
```string
If let urlString = gifUrlString, let gifURL = URL(string: urlString) {

}
```

Reply:
Poza tym pytanie czy chcesz udostepniac ta zmienna czy po prostu mialbys funkcje, ktora to robi. Ta zmienna jesli nie musi byc internal to powinna byc private